### PR TITLE
Us020 suspend respondents account

### PR DESCRIPTION
**Pull request by gemmai95**
Adds functionality for stories US020 and US021 to suspend and re-activate a respondent account. Works with backstage and party.

### CODE REVIEW

**Comment by gemmai95** [comment]
[US021: INT Re-Activate respondents account (L) - Needed for R17](https://trello.com/c/MdXKJfBS/57-us021-int-re-activate-respondents-account-l-needed-for-r17)

File: response_operations_ui/views/reporting_units.py [hunk 1]
```diff
@@ -60,6 +63,10 @@ def view_reporting_unit(ru_ref):
         info_message = f'Response status for {survey["surveyRef"]} {survey["shortName"]}' \
                        f' period {period_arg} changed to {new_status}'
 
+    info = request.args.get('info')
+    if info:
+        info_message = info
```
Comment (andrewmil) [inline response_operations_ui/views/reporting_units.py]: untested line

File: response_operations_ui/views/reporting_units.py [hunk 2]
```diff
@@ -130,3 +137,27 @@ def map_region(region):
         region = "GB"
 
     return region
+
+
+@reporting_unit_bp.route('/confirm-change-respondent-account-status', methods=['GET'])
+@login_required
+def confirm_change_account_status():
+    party_id = request.args['party_id']
```
Comment (andrewmil) [inline response_operations_ui/views/reporting_units.py]: This function is untested

File: response_operations_ui/views/reporting_units.py [hunk 3]
```diff
@@ -130,3 +137,27 @@ def map_region(region):
...
+    respondent = party_controller.get_respondent_details(party_id)
+
+    return render_template('confirm-respondent-account-status.html', form=form, first_name=respondent['respondent_party']['firstName'],
+                           last_name=respondent['respondent_party']['lastName'], change_status=change_status, party_id=party_id, ru_ref=ru_ref)
+
+
+@reporting_unit_bp.route('/change-respondent-account-status')
+@login_required
+def change_account_status():
+    party_id = request.args['party_id']
```
Comment (andrewmil) [inline response_operations_ui/views/reporting_units.py]: This endpoint is also untested

**Pull request closed without merging**